tests/Aspose/Imaging/Tests/Api/TiffApiTest.php

```diff
@@ -196,7 +197,7 @@ public function appendTiffTest()
 
         if (!$this->checkInputFileExists($inputFileName))
         {
-            throw new ArgumentException(
+            throw new \ArgumentException(
                 "Input file " . $inputFileName . " doesn't exist in the specified storage folder: " . $folder . ". Please, upload it first.");
```

Comment:
Is it right syntax ?

Reply:
Yes, "\\" is a namespace separator in PHP, and leading "\\" refers to the global namespace, where _ArgumentException_ is defined.